### PR TITLE
Added JWT authentication backend, provided login on frontend with Redux

### PR DESCRIPTION
**Pull request by slapadominik**
**Back-end changes**:
- added Login method in AuthController
- configured JWT for admin authentication

**Front-end changes**:
- Redux installed, which makes handling authentication easier
- Axios installed, library to make Promise based HTTP requests (generally even easier than Fetch API)

### CODE REVIEW

**Comment by EnjoyTheNoise** [comment]
Good idea to use Axios, also thank you for notifying me in advance about your plans with adding it. :) (it's nice to know about changes earlier than in pr for a change lmao)
Why didn't you use it in Event component tho? I believe it's just simple oversight :)
On the backend everything seems fine.
This time I want somebody else to leave approve tho, more reviews = better quality, right? :D

File: EventAsker.API/Controllers/AuthController.cs [hunk 1]
```diff
@@ -29,6 +34,22 @@ public class AuthController : Controller
...
+                return Unauthorized();
+                
+            AdminLoginDto adminLoginDto = new AdminLoginDto(){
+                AdminId = admin.AdminId,
+                Username = admin.Username
+            };
+
+            string tokenString = _service.BuildToken(adminLoginDto);
+
+            return Ok( new { tokenString });
```
Comment (jskurzynska) [inline EventAsker.API/Controllers/AuthController.cs]: Try sth like that  "new OkObjectResult(tokenString)"

File: EventAsker.SPA/src/reducers/types.js [hunk 2]
```diff
@@ -0,0 +1 @@
+export const SET_CURRENT_USER = 'SET_CURRENT_USER';
```
Comment (jskurzynska) [inline EventAsker.SPA/src/reducers/types.js]: it's not type of reducer - it's action type

File: EventAsker.SPA/src/helpers/authManager.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import axios from 'axios';
+import setAuthorizationToken from './authorizationToken';
+import {BASE_URL} from '../constants';
+import { SET_CURRENT_USER } from '../reducers/types';
+import jwt from 'jsonwebtoken';
+
+export const setCurrentUser = (user) => {
```
Comment (jskurzynska) [inline EventAsker.SPA/src/helpers/authManager.js]: it should be in file userAction or sth like that

**Comment by slapadominik** [comment]
@EnjoyTheNoise, @jskurzynska Thank you for your feedback, I'll make these changes soon.

**Comment by slapadominik** [comment]
Done. Is it fine now?

**Pull request merged**